ecosystem/sep-0012.md

```diff
@@ -6,17 +6,13 @@ Title: Anchor/Client customer info transfer
 Author: Interstellar
 Status: Active
 Created: 2018-09-11
-Updated: 2018-09-11
-Version 1.1.0
+Updated: 2020-04-06
+Version 1.1.1
 ```
 
-## Deprecation warning
-
-We're currently [recommending](sep-0006.md#recommendations) that anchors and wallets use the SEP-6 interactive flow for transmitting KYC information, rather than SEP-12. Therefore, SEP-12 is **deprecated** for anyone using it as part of a [SEP-6](sep-0006.md) implementation.
-
 ## Abstract
 
-This SEP defines a standard way for stellar wallets to upload KYC (or other) information to anchors who need it.
+This SEP defines a standard way for stellar wallets to upload KYC (or other) information to anchors that implement non-interactive SEP-6.
```

Comment:
Now that SEP-6 is active for non-interactive flows, SEP-12 should be used to transfer KYC/AML information between service and anchors.

Reply:
I'm not sure we want to tie this directly to sep-6 as it can be used outside of it, but i think this is fine for now.  If we start to use sep-12 more we can loosen this up, imo always better to keep things too strict and loosen than the opposite.

Reply:
Okay, thanks. Good point! SEP-12 will probably also be interesting for SEP-26 because there is currently no non-interactive KYC/AML transfer defined there yet.